src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -132,7 +133,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     $this->hardCopy($originDir, $targetDir);
                 }
             }
+            if (is_dir($targetDir)) {
```

Comment:
instead of adding this condition, move this inside the previous `if`. There are 2 reasons for this:

- the variable is not assigned if the previous condition was false, which means it will either reuse the value from the previous iteration or fail with an undefined variable notice. Both are bad behaviors
- if we decided to install some assets for the bundle (which is what the previous condition is about), we know that the director us valid